ci-operator/config/rhmdnd/compserv/rhmdnd-compserv-main.yaml

```diff
@@ -0,0 +1,23 @@
+binary_build_commands: go build -o builds/ cmd/compserv-server.go
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
I think it should be fine to start with something more modern.

Reply:
Addressed this in compserv https://github.com/rhmdnd/compserv/pull/47